internal/test/e2e/cluster_test.go

```diff
@@ -1110,41 +1110,22 @@ func TestEphemeralAccounts(t *testing.T) {
 		t.SkipNow()
 	}
 
-	// Create cluster
+	// create cluster
 	cluster := newTestCluster(t, testClusterOptions{hosts: 1})
 	defer cluster.Shutdown()
 	tt := cluster.tt
 
-	// Shut down the autopilot to prevent it from interfering.
+	// shut down the autopilot to prevent it from interfering with the test
 	cluster.ShutdownAutopilot(context.Background())
 
-	// Wait for contract and accounts.
-	contract := cluster.WaitForContracts()[0]
-	accounts := cluster.WaitForAccounts()
-
-	// Shut down the autopilot to prevent it from interfering with the test.
-	cluster.ShutdownAutopilot(context.Background())
-
-	// Newly created accounts are !cleanShutdown. Simulate a sync to change
-	// that.
-	for _, acc := range accounts {
-		if acc.CleanShutdown {
```

Comment:
The clean shutdown flag didn't change right? So we can still verify that somewhere?

Reply:
The problem is that you are racing the sync that now occurs when opening the account, which flips it back. Perhaps adding a small unit test that verifies these flags on a fresh account is the way to go. I'll re-add it somehow